custom/icds_reports/models/aggregate.py

```diff
@@ -563,6 +565,22 @@ class Meta(object):
     _agg_atomic = False
 
 
+class AggregateTHRForm(models.Model, AggregateMixin):
```

Comment:
This new model is added. Do I need to do some special step to create this in citus machines or this will be taken by the deploy? @emord  @calellowitz

Reply:
no, the django migrations will create it for you